src/edutap/wallet_apple/plugins.py

```diff
@@ -57,9 +75,29 @@ def get_logging_handlers() -> list[Logging]:
     plugins = [
         ep.load() for ep in eps if ep.name.startswith("Logging")
     ] + _PLUGIN_REGISTRY.get("Logging", [])
-    # if not plugins:
-    #     raise NotImplementedError("No logging plug-in found")
     for plugin in plugins:
         if not isinstance(plugin, Logging):
             raise ValueError(f"{plugin} not implements Logging")
     return [plugin() for plugin in plugins]
+
+
+def get_dynamic_settings_handlers() -> list[DynamicSettings]:
+    eps = entry_points(group="edutap.wallet_apple.plugins")
+    # allow multiple entries by searching for the prefix
+    plugins = [
+        ep.load() for ep in eps if ep.name.startswith("DynamicSettings")
+    ] + _PLUGIN_REGISTRY.get("DynamicSettings", [])
+    for plugin in plugins:
+        if not isinstance(plugin, DynamicSettings):
+            raise ValueError(f"{plugin} not implements DynamicSettings")
+    return [plugin() for plugin in plugins]
+
+
+def get_dynamic_settings_handler() -> DynamicSettings | None:
```

Comment:
We should think about iterating the plugins and try to query dynamic settings from each, first one returning settings by criteria wins. What do you think?